apps/backend/src/api/export_import.py

```
@@ -0,0 +1,785 @@
+        )
+        recent_imports_result = await session.execute(recent_imports_query)
+        recent_imports = [
+            ConfigurationImportResponse.from_orm(import_op)
+            for import_op in recent_imports_result.scalars().all()
+        ]
+
+        # Calculate archive statistics
+        total_archive_size_query = select(
+            func.count(ConfigurationExport.id).label("total_archives"),
+            func.coalesce(func.sum(ConfigurationExport.archive_size_bytes), 0).label("total_size"),
+            func.count().filter(ConfigurationExport.encrypted == True).label("encrypted_count"),
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `ConfigurationExport.encrypted:` for truth checks

Replace with `ConfigurationExport.encrypted`

(E712)